geoclue/geoclue/gc-provider.c

```diff
@@ -196,9 +196,27 @@ gc_provider_remove_client (GcProvider *provider, const char *client)
 	return TRUE;
 }
 
+static gboolean 
+gc_provider_remove_client (GcProvider *provider, const char *client)
+{
+	int *pcount;
+	GcProviderPrivate *priv = GET_PRIVATE (provider);
+	
+	pcount = g_hash_table_lookup (priv->connections, client);
+	if (!pcount) {
```

Comment:
Minor detail, but the returned value is not actually getting used for anything. This could save a variable and couple of lines by checking instead what g_hash_table_remove() returns. (And there would have been g_hash_table_contains too.

Reply:
Good point, I've changed that.